base/beamer.cls

```diff
@@ -302,7 +302,9 @@
 \newlength\beamer@paperheight
 \setlength\beamer@paperheight{9.60cm}
 
+\def\insertaspectratio{43}%
 \DeclareOptionBeamer{aspectratio}[43]{%
```

Comment:
I ask this question without much knowledge of LaTeX, but would it make sense to avoid the duplication of "43"? I'm guessing there is a way, but that you decided that the increase in code complexity is not worth it? (no need for a detailed reply, feel free to ignore)

Reply:
... one could store the default value in a variable, but given the maturity of the beamer class, it is unlikely to change.

Reply:
Makes sense, thank you!

Reply:
You're welcome!